File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/F. Multiplication table/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/F
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short n;
    cin >> n;
    for(int i=1; i <= 12; i++){
        printf("%d * %d = %d\n", n, i, (n*i));
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/F. Multiplication table/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/F

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemF
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            for(int i=1; i <= 12; i++){
                Console.WriteLine($"{n} * {i} = {(n*i)}");
            }
        }
    }
}
